autogen/Hardware.m

```diff
@@ -0,0 +1,20 @@
+classdef Hardware
```

Comment:
Do you think `TargetHardware` is a better name?

Reply:
The way I view it is they are just hardware, and whatever you select for your model is hardware you are targeting. Essentially my view is using target in regards to the user, not us as developers of the project.

Reply:
Fair. My thoughts were that `Hardware` is pretty generic, but `TargetHardware` also suffers from the same generic problem. Ultimately not a big deal.